example/hetzner-dedi-nvme-zfs.nix

```
@@ -0,0 +1,88 @@
+# disko config for Hetzner Online (dedicated)
+# Developed and tested on AX41-NVMe (legacy BIOS) in August 2023.
+
+# Matching grub config:
+#
+# boot.initrd.availableKernelModules = [ "nvme" ];
+# boot.loader.grub = {
+#   devices = [ "/dev/nvme0n1" "/dev/nvme1n1" ];
+#   efiSupport = true;
+#   efiInstallAsRemovable = true;
+# };
+# fileSystems."/" = {
+#   device = "tank/root";
+#   fsType = "zfs";
+#   neededForBoot = true;
+# };
+#
+# Migrate your network config from the rescue system.
+#
+# Some settings might be not optimal or superfluous.
+
+{ lib, disks ? [ "/dev/nvme0n1" "/dev/nvme1n1" ], ... }: {
+  disk = lib.genAttrs disks (dev: {
+    device = dev;
+    type = "disk";
+    content = {
+      type = "table";
```

We no longer use type = "table" for new things.

```suggestion
      type = "gpt";
```